Software Engineer Big Data.md

```diff
@@ -1,23 +1,14 @@
-[HolidayCheck](http://www.holidaycheck.de/) is deeply committed to providing the best experience in online travel for our customers.
-Help our growing team of data scientists and engineers to unleash the potential of billions of events every day to provide 25 million travellers every month with intelligent optimizations as a ...
-
-# (Senior) Software Engineer – Big Data
-
-## Your Tasks
-- Design, model, implement and maintain Hadoop solutions supporting raw-event collections
 - Help internal clients with pragmatic strategic & ad-hoc data requirements
-- Optimize our data infrastructure to become a reliable, tested real-time solution
 - Work closely with data scientists and engineers to build personalized experiences for our users
 - Drive research initiatives to innovate and support our data driven approach
-- Perform data extraction, aggregation from multiple sources, includng Google Analytics
+- Perform data extraction, aggregation from multiple sources, includng Google Analytics / GoogleBigQuery
 - Create confidence in our data by tests, automation, monitoring and neat visualizations
 
 ## Your Profile
-- Several years of real-world experience with Apache Hadoop (with e.g. HDFS, Hive, HBase, Pig)
-- Experienced software engineer, proficient with several languages (we use Scala a lot)
-- Good command of several scripting languages, like Python, Bash, JavaScript or even R
-- Proficient with SQL (MySQL, MS-SQL are running) and NoSQL (e.g. MongoDB, Neo4j, Cassandra) databases and querying them
-- Sysadmin experience (Linux, AWS, Docker) is a plus
+- You are eager to learn new technologies
+- You are able to transform solutions into code (Scala is a plus)
```

Comment:
Personally prefer: transform ideas into code.

The code would be the solution itself.